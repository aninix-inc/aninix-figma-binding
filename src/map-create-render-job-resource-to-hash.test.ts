import { expect, test } from 'bun:test'
import { mapCreateRenderJobResourceToHash } from './map-create-render-job-resource-to-hash'

test('mapCreateRenderJobResourceToHash', async () => {
  const result = await mapCreateRenderJobResourceToHash({ test: 'object' })
  expect(result).toEqual(
    'f5385eaa9068c121e39b71e8bce648628eb0ff7b3b106bb8395a5f74f3c23de0'
  )
})
