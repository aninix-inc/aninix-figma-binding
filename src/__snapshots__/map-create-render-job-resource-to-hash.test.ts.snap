// Bun Snapshot v1, https://goo.gl/fbAQLP

exports[`mapCreateRenderJobResourceToHash 1`] = `"f5385eaa9068c121e39b71e8bce648628eb0ff7b3b106bb8395a5f74f3c23de0"`;
